task_01.rst

```diff
@@ -0,0 +1,110 @@
+#################
+Why I Love Python
+#################
+
+Why I love **Python**, a language worth knowing. 
+
+**************
+Running It
+**************
+Python synthax is really quite simple to execute. Anyone can do it just by executing the *python* command on the command line and invoking the interpreter. 
+
+You can also execute python programs by calling the program filename such as:
+
+.. code-block:: console
+
+       me@mymachine$ python my_file_name.py
+
+*****************
+The Syntax
+*****************
+
+Python syntax is very easy to write. Take a look at this simple statement.
+
+.. code-block:: python
+
+      >>> print 'Hello!'
+
+*******************
+The People
+*******************
+
+It's amazing how nice people are in the Python development world. Just as with any 
+community, however, it's important to follow certain rules of decorum. 
+
+       - Be courteous to others
+       - Always read the manual before asking questions
+       - Be succinct in your questions
+
+                   - Don't give unnecessary details if they aren't needed
+                   - Don't start a question with an apology or an announcement that you are 
+                      going to ask a question. 
+
+                      *Just ask the question!*
+
+********************
+The Tools
+********************
+
+Python has a ton of excellent tools available for use. 
+
+$$$$$$$$$$$$$$$$
+Docutils
+$$$$$$$$$$$$$$$$
+
+A fantastic documentation generator that interprets reStructuredText and turns it into
+rich documents. 
+
+$$$$$$$$$$$$$$$$$$$$$
+Web Frameworks
+$$$$$$$$$$$$$$$$$$$$$
+
+Python has many great web frameworks, only a few of which  will be listed here. 
+
+&&&&&&&&&&
+Django
+&&&&&&&&&&
+
+One of the world's leading content management systems for the web. To learn more 
+about Django, visit:
+
+https://www.djangoproject.com/
+
+&&&&&&&&&&&&&&&
+Pyramid
+&&&&&&&&&&&&&&&
+
+A relative newcomer to the web framework. Pyramid began as part of the Pylons 
+project and has an interesting(though challenging) templating language. 
+
+******************
+Other Details
+******************
+
+1. Python can be used for almost any task you can think of.
+2. It has a huge library of available modules. 
+3. Pip extends that library even more.
+4. Why aren't you using it already! Go write some Python!
+
+.. table:: Sources
+
+        ======    =======
+        Name       Contact
+        ======    =======
+        Me             me@mysite.com
+        Myself     myself@me.com
+         I                  i@mii.com
+        ======    =======
+
+
+.. note:: This is just a little taste of how great Python can be!
+
```

Comment:
Too much extra whitespace here at the end. Try to be a bit zealous in checking that in the future!